.reflame.config.jsonc

```diff
@@ -273,16 +268,9 @@
   "libraries": {
     "@highlight-run/ui": {
       "sourceDirectory": "packages/ui/src",
-      "entryPoints": {
-        "/components": "/components/index.ts",
-        "/css": "/css.ts",
-        "/keyframes": "/keyframes.ts",
-        "/sprinkles": "/sprinkles.ts",
-        "/vars": "/vars.ts",
-        "/theme": "/theme.ts",
-        "/colors": "/colors.ts",
-        "/borders": "/borders.ts",
-        "/styles.css": "/noop.ts",
+      "exports": {
+        "./components": "./components/index.ts",
+        "./styles.css": "./noop.ts",
```

Comment:
We need to override components because the dist path points to `dist/components.js` which is not a 1:1 mapping to the source modules, and styles.css because we're using our custom css bundle.